modules/monitoring-troubleshooting-remote-write-configuration.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/troubleshooting-monitoring-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting-remote-write-configuration_{context}"]
+= Troubleshooting remote write configuration
+
+If your remote write configuration does not work properly, you can verify the running configuration from the Prometheus API and inspect metrics to discover other possible issues.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa-hcp,openshift-rosa[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+endif::openshift-dedicated,openshift-rosa-hcp,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa-hcp,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-dedicated,openshift-rosa-hcp,openshift-rosa[]
+* You have configured remote write storage.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Verify the running remote write configuration from the Prometheus API:
+// tag::CPM[]
+
+** To verify remote write configuration for core platform monitoring:
++
+[source,terminal]
+----
+$ oc exec prometheus-k8s-0 -c prometheus -n openshift-monitoring -- curl -s 'http://localhost:9090/api/v1/status/config' 
+----
+** To verify remote write configuration for user workload monitoring:
+// end::CPM[]
++
+[source,terminal]
+----
+$ oc exec prometheus-user-workload-0 -c prometheus -n openshift-user-workload-monitoring -- curl -s 'http://localhost:9090/api/v1/status/config' 
+----
++
+.The formatted example output
+[source,terminal]
+----
+...
+remote_write:
+- url: https://remote-write-endpoint.example.com
+  remote_timeout: 30s
+  write_relabel_configs:
+  - separator: ;
+    target_label: __tmp_openshift_cluster_id__
+    replacement: 0b02e767-c309-41e9-8727-03bb50f0fc89
+    action: replace
+  - separator: ;
+    regex: __tmp_openshift_cluster_id__
+    replacement: $1
+    action: labeldrop
+  protobuf_message: prometheus.WriteRequest
+  authorization:
+    type: Bearer
+    credentials: <secret>
+  follow_redirects: true
+  enable_http2: true
+  queue_config:
+    capacity: 10000
+    max_shards: 50
+    min_shards: 1
+    max_samples_per_send: 2000
+    batch_send_deadline: 5s
+    min_backoff: 30ms
+    max_backoff: 5s
+  metadata_config:
+    send: true
+    send_interval: 1m
+    max_samples_per_send: 2000
+...
+----
++
+[NOTE]
+====
+The formatted example output uses a filtering tool, such as `jq`, to provide the formatted indented JSON. See the link:https://stedolan.github.io/jq/manual/[jq Manual] (jq documentation) for more information about using `jq`.
```

Comment:
the paragraph is fine, just FYI: use a jq tool only will not get a formatted indented JSON, example https://privatebin.corp.redhat.com/?35bf7440a03eabfa#5uhjxvXRfRSD9gubPgWWAjL5Tr1Yk6UsyNmTKeThhsE2, need to replace `\n` to new line for the data.yaml part, then can get  a formatted indented JSON